IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/test/src/server/simple_tcp_server.c

```diff
@@ -0,0 +1,92 @@
+/* wolf_server.c
```

Comment:
File name should be "simple_tcp_server.c"

Reply:
Thanks. Fixed